handlers/chikka/chikka_test.go

```diff
@@ -90,7 +90,7 @@ var defaultSendTestCases = []ChannelSendTestCase{
 		Text: "Simple Message", URN: "tel:+63911231234",
 		ResponseToID:         5,
 		ResponseToExternalID: "external-id",
-		ResponseBody:         `{"status":400,"message":"BAD REQUEST","description":"Invalid/Used Request ID"}`,
+		ResponseBody:         `{"status":400,"message":"BAD REQUEST","description":"Invalid\\/Used Request ID"}`,
```

Comment:
Does the raw text actually look like `Invalid\/Used Request ID`?

Reply:
The python code seems to use `Invalid/Used Request ID`

Reply:
I got this by looking at the channel log